src/testSession.ts

```diff
@@ -334,7 +334,9 @@ export class TestSession<T extends TestSessionOptions = TestSessionOptions> exte
           throw new Error(`${executable} executable not found for creating scratch orgs`);
         }
 
-        let baseCmd = `sf org:create:scratch --json -y ${org.duration ?? '1'} -w ${org.wait ?? 60}`;
+        let baseCmd = `sf org:create:scratch --name 'cli-plugins-testkit' --json -y ${org.duration ?? '1'} -w ${
+          org.wait ?? 60
+        }`;
```

Comment:
This could be considered a breaking change since the command flag will override config file settings.  It's unlikely that consumers are using the org name in their tests but if they are this would break them.  Should we:
1. bump major version?
2. use a different field that is also queryable?
3. some other option?

Reply:
oh right, yeah nvm I see any other available field that isn't also a breaking change.